nginx/etc/nginx/conf.d/default.conf

```diff
@@ -2,12 +2,17 @@
 resolver 127.0.0.11;
 
 # Select upstream backend to use based on presense of Xdebug cookies and Blackfire headers
-map "$http_X_BLACKFIRE_QUERY:$cookie_XDEBUG_SESSION$cookie_XDEBUG_PROFILE$cookie_XDEBUG_TRACE$arg_XDEBUG_SESSION$arg_XDEBUG_SESSION_START" $fastcgi_backend {
+map "$http_X_BLACKFIRE_QUERY:$cookie_XDEBUG_SESSION$cookie_XDEBUG_PROFILE$cookie_XDEBUG_TRACE$arg_XDEBUG_SESSION$arg_XDEBUG_SESSION_START:$cookie_SPX_ENABLED$cookie_SPX_KEY$arg_SPX_ENABLED$arg_SPX_KEY$arg_SPX_UI_URI" $fastcgi_backend {
+    volatile;
```

Comment:
@navarr  I was having issues with cookies not being respected; however, it could have been because of Varnish caching the requests. This line shouldn't need to be there.